integration/k8/README.md

```diff
@@ -69,9 +69,16 @@ approach is preferred if the host OS already provides the GEOPM Access Service
 or if installing the GEOPM packages on the host is preferred over running a
 privileged container.
 
-To enable this, edit the GEOPM systemd unit file using the `systemctl edit
-geopm` command. Modify the `ExecStart` field to include the `--grpc` option and
-set the `Type` to `simple`:
+A prerequisite for using the host OS for GEOPM access is to build the install
+the `geopmd-proxy` Rust binary in the system path. On debian based systems,
+packaging the `geopmd-proxy` file is enabled with the `cargo deb` command, but
+there is currently no support for RPM based packaging of the `geopmd-proxy`
+staticly linked binary executable. See the `geopmdrs/README.md` file for more
```

Comment:
Discussion of lack of RPM support buried in this README.  IMO this should be in the top level README.